### PR TITLE
Data Science with Python : Usage of get_dummies to create Dummy Variables with Pandas.

### LINKED ISSUE #3176: Data Science with Python : Usage of get_dummies to create Dummy Variables with Pandas
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Usage of get_dummies to create Dummy Variables with Pandas Tutorial to Usage of get_dummies to create Dummy Variables with Pandas and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Machine Learning

Mentors Assigned : Dhruv Bajaj / Kareem Negm

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by deepthi1107**
<hr>

## Description 📜

Audio explaining Usage of get_dummies to create Dummy Variables with Pandas.
Fixes #3176

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)

<hr>

## Domain of Contribution 📊

- [x] Datascience with Python

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- *Label encoder:* Label Encoding refers to converting the labels into a numeric form so as to convert them
+into the machine-readable form.
+
+- *One- hot encoding:* A one hot encoding is a representation of categorical variables as binary vectors. 
+This first requires that the categorical values be mapped to integer values.
+
+
+
+![Capture_3](https://user-images.githubusercontent.com/79050917/135568985-3e949ce7-5aba-46a6-9cf2-db6146f1309a.PNG)
```
Comment (DhruvBajaj01) [inline Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md:20]: Please remove code images from repo

File: Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+**Adavtages:**
+- Converts the categorical data to numeric which makes easily understandable by ml algorithms.
+
```
Comment (DhruvBajaj01) [inline Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md:24]: Add Python libraries used

**Comment by SaloniThete** [comment]
Name: Saloni Thete
Sr.No: 821
Type: Documentation
Approach: Explanation with detail including python libraries used.
@prathimacode-hub,kindly assign :)

**Comment by deepthi1107** [comment]
I have made changes @DhruvBajaj01

File: Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import pandas as pd
+
+pd.get_dummies()
+
+**Adavtages:**
+- Converts the categorical data to numeric which makes easily understandable by ml algorithms.
+
+**Disadvantage:**
+- Might impact on size if it has more categories in the dataset which results in increasing the complexity.
+
```
Comment (DhruvBajaj01) [inline Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md:35]: Add jupyter notebook for the same

Comment (deepthi1107) [reply Datascience_With_Python/Machine Learning/Audios/Usage of get_dummies to create Dummy Variables with Pandas/README.md:35]: Ok @DhruvBajaj01

**Comment by prathimacode-hub** [comment]
> Name: Saloni Thete Sr.No: 821 Type: Documentation Approach: Explanation with detail including python libraries used. @prathimacode-hub,kindly assign :)

For assignments, you have to look through issues not PR. @SaloniThete

**Comment by DhruvBajaj01** [comment]
Have you added an audio file?

**Comment by deepthi1107** [comment]
Yes @DhruvBajaj01 in readme.md file.

**Review by DhruvBajaj01 — COMMENTED** [review]
Reviewed Audio file! nice work @deepthi1107 ! @prathimacode-hub  please go through the PR for final review and suggest changes if any.

**Review by prathimacode-hub — APPROVED** [review]
PR approved. Keep going high. ✨✔@deepthi1107

**Pull request merged**